.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+on:
+  push:
+    tags:        
+      - 'v*'
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+  npm-publish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build
+      # Bump the version in package.json
```

Comment:
What's this comment for?

Reply:
Just to explain what exactly the step after does, in case someone viewing the file is unfamiliar with the npm version command